Ubuntu/master.sh

```diff
@@ -5,26 +5,13 @@ if [ "$EUID" -ne 0 ]; then
   exit
 fi
 
-# Mark packages on hold to avoid auto upgrade.
-apt-mark hold kubelet
-apt-mark hold kubectl
-apt-mark hold kubeadm
-apt-mark hold containerd.io
-apt-mark hold docker-buildx-plugin
-apt-mark hold docker-ce
-apt-mark hold docker-cli
-apt-mark hold docker-ce-rootless-extras
-apt-mark hold docker-compose-plugin
-apt-mark hold snapd
-apt-mark hold systemd
-apt-mark hold systemd-sysv
-apt-mark hold systemd-timesyncd
-
 TIMEOUT=300
 SLEEP_INTERVAL=1
 
+# Remove the older packages.
```

Comment:
```suggestion
# Remove all older packages.
```

We should be a bit more intelligent and remove them only if they are not the expected version. We should ensure that this script is idempotent where if we run it multiple times, we don't do wasted work.

Reply:
We can add additional check before remove in separate PR.